questions/qa-mono-multilingual.zh-hans.html

```diff
@@ -0,0 +1,148 @@
+f.authors = 'Richard Ishida, W3C' // author(s) and affiliations
+f.previousauthors = '' // as above
+f.modifiers = '' // people making substantive changes, and their affiliation
+f.searchString = 'qa-mono-multilingual' // blog search string - usually the filename without extensions
+f.firstPubDate = '2003-08-12' // date of the first publication of the document (after review)
+f.lastSubstUpdate = { date:'2004-04-30', time:'14:55'}  // date and time of latest substantive changes to this document
+f.status = 'published'  // should be one of draft, review, published, notreviewed or obsolete
+f.path = '../' // what you need to prepend to a URL to get to the /International directory 
+
+// AUTHORS AND TRANSLATORS should fill in these assignments:
+f.thisVersion = { date:'2021-04-27', time:'14:07'} // date and time of latest edits to this document/translation
+f.contributors = '' // people providing useful contributions or feedback during review or at other times
+// also make sure that the lang attribute on the html tag is correct!
+f.sources = '' // describes sources of information
+
+// TRANSLATORS should fill in these assignments:
+f.translators = '<a href="https://github.com/international-explore" title="Huan Cui - international-explore">Huan Cui - international-explore</a>' // translator(s) and their affiliation - a elements allowed, but use double quotes for attributes
+
+f.breadcrumb = 'other'
+
+f.additionalLinks = ''
+</script>
+<script src="qa-mono-multilingual-data/translations.js"> </script>
+<script src="../javascript/doc-structure/article-dt.js"> </script>
+<script src="../javascript/boilerplate-text/boilerplate-zh-hans.js"></script>
+<!--TRANSLATORS must change -en above to the subtag for their language!-->
+<script src="../javascript/doc-structure/article-2022.js"> </script>
+<script src="../javascript/articletoc-2022.js"></script>
+<link rel="stylesheet" href="../style/article-2022.css">
+<link rel="copyright" href="#copyright">
+</head>
+
+<body>
+<header>
+  <nav id="mainNavigation"></nav><script>document.getElementById('mainNavigation').innerHTML = mainNavigation</script>
+
+  <h1>单语言和多语言网站</h1>
+</header>
+
+
+<section id="sidebarExtras">
+  <section>
+    <h2 id="links" class="notoc"><a href="#links">相关链接</a></h2>
+    <p><a class="print" href="/International/questions/qa-international-multilingual">单语言和多语言网站</a></p>
+  </section>
+</section>
+
+
+<section id="question">
+  <div id="audience">
+    <!--p><span id="intendedAudience" class="leadin">Intended audience:</span>
+    anyone who is wondering about the difference between monolingual and multilingual sites.</p-->
+    <div id="updateInfo"></div>
+  <script>document.getElementById('updateInfo').innerHTML = g.updated</script>
+    </div>
+
+
+
+
+  <h2>问题</h2>
+    <p class="question">国际化网站如何在单语言和多语言之间取舍？</p>
+  </section>
+
+
+
+
+
+<h2>回答</h2>
+  <p>国际化网站为全球用户提供服务。不仅要考虑使用单语言还是多语言的方式外，还需要决定是否为全球用户做内容适配。</p>
```

Comment:
```suggestion
  <p>国际化网站为全球用户提供服务。除了要考虑使用单语言还是多语言的方式外，还需要决定是否为全球用户做内容适配。</p>
```

「除了……还」「不仅……而且」感觉一般是这么对应的。